actions/check-git-tag-for-monorepo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "check-git-tag-for-monorepo",
-  "version": "0.1.0",
```

Comment:
Don't bump the version here directly, use `pnpm changeset add`.

Reply:
Hi Erik, thanks for the comment, I've corrected it. I suppose the changeset file once eventually merged will trigger the cicd-changeset action which takes care of versioning?

Reply:
Yes. Well merge the “Version packages” PR which will publish tags.